TKGame/TKGame.cs

```diff
@@ -42,9 +43,11 @@ public class TKGame : Game
         // TODO: Refactor out of the main TKGame class
         private static string currentStageName = "auto_saved_stage_data_5" + ".json";
         Stage currentStage;
+        Stage leftStage;
+        Stage rightStage;
```

Comment:
Will have to be factored into the Stage's ownership later on. It will have a list of the adjacent stages you can move to from the current stage. This is fine for now